projects/storefrontstyles/scss/components/product/search/_searchbox.scss

```diff
@@ -296,6 +296,17 @@
         z-index: 20;
         padding-top: 25px;
       }
+
+      @include forFeature('a11yActiveAndFocusedSearchBoxComponent') {
+        &:focus-within,
+        &:active {
```

Comment:
The outline should be visible when the keyboard is used for navigation only. Wouldn't this selector mean it is visible even when mouse is used? Maybe we should mimic how other inputs in the app work?

Reply:
I've just updated it.